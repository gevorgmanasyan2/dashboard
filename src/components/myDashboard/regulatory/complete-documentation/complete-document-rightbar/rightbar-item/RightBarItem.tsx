import React, {FC} from 'react'

import Title from '../../../../../title/Title'

import './rightBarItem.scss'

interface RightBarItemProps {
  title: string
}

const RightBarItem: FC<RightBarItemProps> = ({title, children}) => {
  return (
    <div className='rightBar__item'>
      <div className='rightBar__item__header'>
        <Title titleText={title} level={5}/>
      </div>
      {children}
    </div>
  )
}

export default RightBarItem