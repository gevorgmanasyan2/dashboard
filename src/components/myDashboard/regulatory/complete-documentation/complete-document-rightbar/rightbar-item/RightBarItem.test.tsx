/* eslint-disable no-undef */
import React from 'react'
import { shallow } from 'enzyme'
import RightBarItem from './RightBarItem'

it('should render RightBarItem component', () => {
  const mockItem = {
    title: 'rightBar',
    children: <div></div>,
  }
  expect(shallow(<RightBarItem {...mockItem} />)).toMatchSnapshot()
})
