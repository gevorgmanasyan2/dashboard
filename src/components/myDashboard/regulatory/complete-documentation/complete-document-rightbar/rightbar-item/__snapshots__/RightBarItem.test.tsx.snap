// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render RightBarItem component 1`] = `
<div
  className="rightBar__item"
>
  <div
    className="rightBar__item__header"
  >
    <Title
      level={5}
      titleText="rightBar"
    />
  </div>
  <div />
</div>
`;
